datahub-upgrade/src/main/resources/logback.xml

```diff
@@ -0,0 +1,81 @@
+<configuration scan="true" scanPeriod="120 seconds">
+    <property name="LOG_DIR" value="${LOG_DIR:-/tmp/datahub/logs/gms}"/>
```

Comment:
Can we update this LOG_DIR default to point to `/tmp/datahub/logs/datahub-upgrade`?

Reply:
done